### PR TITLE
Todos functionality

### PR DESCRIPTION
**Pull request by abel-tefera**
### Summary

- Created classes to hold todos data
- Created custom element for the todo items
- Used local storage to store todo data

### CODE REVIEW

File: src/modules/Todos.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          ...todo,
+          completed: !todo.completed,
+        };
+      }
+      return {
+        ...todo,
+      };
+    });
+    localStorage.setItem('todoData', JSON.stringify(this.todoList));
+  }
```
Comment (omarsalem7) [inline src/modules/Todos.js]: - [ ] Great work for creating the addTodo function but you create removeCompleted, and markCompleted functions but these methods don't need for this milestone. You should add other functions removeTodo and editTodo as stated by the requirements and also make sure to include other cases as in the image below
>image

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @abel-tefera 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- AddTodo method works properly ✔️
- Data saved in local storage ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/Todos.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    todo.description = newDesc;
+    this.todoList = this.todoList.map((todos) => {
+      if (todos.index === idx) {
+        return todo;
+      }
+      return {
+        ...todos,
+      };
+    });
+  }
```
Comment (thecodechaser) [inline src/modules/Todos.js]: - [ ] The edit functionality is not working as expected, If I edit any task and refresh the page I still get the old task description it's not being updated in the local storage. Please make sure to fix this issue and users should be able to get the edited task description after the page refresh.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @abel-tefera👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- Highly organized code. ✔️
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- No linter errors. :heavy_check_mark:



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @abel-tefera, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**